source/slides/13_ansible.rst

```diff
@@ -0,0 +1,520 @@
+.. _13_ansible:
+
+Ansible
+=======
+
+What is Ansible?
+----------------
+
+* Started in 2012
+* More than 600 contributors
+* Orchestration Engine
+
+  * Configuration Management
+  * Application Deployment
+  * Continuous Delivery
+
+Ansible Commands
+----------------
+
+.. rst-class:: build
+
+``ansible``
+  Low level command to execute simple Ansible resources
+``ansible-playbook``
+  Tool to run Ansible Playbooks
+``ansible-pull``
+  Tool to pull remote Ansible configurations and run the playbooks
```

Comment:
Revise to:
    Tool to pull remote Ansible configurations and run the playbooks locally.